api-reference/beta/api/connector_post_memberof.md

```diff
@@ -1,6 +1,6 @@
-# Add Connector to connectorGroup
+# Add Connector to connectorGroups
 
-Use this API to add a connector to a new connectorGroup.
+Use this API to add a connector to a new connectorGroups.
```

Comment:
Again the original is mostly connect

Title should be:  Add a connector to a connectorGroup
 For this text, the original is correct.